proxy/backend/core/src/main/java/org/apache/shardingsphere/proxy/backend/handler/distsql/ral/updatable/ImportDatabaseConfigurationUpdater.java

```diff
@@ -33,6 +33,8 @@
  */
 public final class ImportDatabaseConfigurationUpdater implements RALUpdater<ImportDatabaseConfigurationStatement> {
     
+    private final YamlDatabaseConfigurationImportExecutor databaseConfigImportExecutor = new YamlDatabaseConfigurationImportExecutor();
```

Comment:
No other function uses databaseConfigImportExecutor , can it be a local variable?

Reply:
The validateHandler in YamlDatabaseConfigurationImportExecutor needs a mock, and local variables seem to be unsolvable.

Reply:
Well.